### ISSUE TITLE
Cosmos cannot find Vite configuration file with "ts" extension

### ISSUE THREAD

**Issue by max-scopp** [react-cosmos/react-cosmos#1511]
Using Cosmos in combination with the Vite bundler and the following configuration:

```json
{
  "plugins": ["react-cosmos-plugin-vite"]
}
```

Cosmos is unable to locate the Vite configuration file as it seems to expect a "js" extension instead of the "ts" extension. When using TypeScript, Vite suggests using the "ts" extension for their configuration file.

**Steps to Reproduce:**

1. Create a Vite project using the command: `yarn create vite my-vite-app`
2. Select a framework (e.g., React) when prompted.
3. Choose the TypeScript variant.
4. Install Cosmos according to the official documentation.
5. When starting Cosmos with the command `yarn cosmos`, the following message is logged: "No vite config found, using default settings."

**User Info:**

```yaml
"react-cosmos@npm:next":
  version: 6.0.0-beta.1
  resolution: "react-cosmos@npm:6.0.0-beta.1"
```

**Console Output:**

```
[Cosmos] Using cosmos config found at cosmos.config.json
[Cosmos] Found 1 plugin: Vite
[Cosmos] Port 5000 already in use, trying next...
[Cosmos] See you at http://localhost:5001
[Cosmos] No vite config found, using default settings
```

**Cosmos Configuration:**

```json
// cosmos.config.json
{
  "plugins": ["react-cosmos-plugin-vite"]
}
```

**Vite Configuration:**

```typescript
// vite.config.ts
import { defineConfig } from "vite";
import react from "@vitejs/plugin-react";
import path from "path";
import kebabCase from "kebab-case";

export default defineConfig({
  plugins: [
    react({
      jsxRuntime: "classic"
    })
  ],
  css: {
    preprocessorOptions: {
      scss: {
        additionalData: ""  // omitted per company guidelines
      }
    },
    modules: {
      generateScopedName: (name, filename) => {
          // omitted per company guidelines
      },
      localsConvention: ""  // omitted per company guidelines
    }
  },
  build: {
    emptyOutDir: true,
    lib: {}  // omitted per company guidelines, no special magic though
    rollupOptions: {} // omitted per company guidelines, no special magic though
  }
});
```

**Additional Context:**

No additional context is available at the moment.

**Comment by skidding** [comment]
@max-scopp Pushed a fix in #1512.

Can you please install `react-cosmos-plugin-vite@canary` and confirm it works as expected for you? I'll release a new beta version afterwards.

**Comment by max-scopp** [comment]
Thank you, @skidding, for the fix in #1512. I have installed `react-cosmos-plugin-vite@canary` and can confirm that it resolves the issue. I have successfully tested it, and it now works as expected.

Thank you for your quick response and the exceptional work you provide.

**Comment by max-scopp** [comment]
For the benefit of future readers, when using the fixed version, I had to **update both** `react-cosmos` and `react-cosmos-plugin-vite` to their respective canary versions.

In my case, the control panel would be rendered without any functionality.

**Comment by skidding** [comment]
@max-scopp Thanks for the heads-up!

Just published `6.0.0-beta.2` for all packages.
